web/src/views/ProfileEditable.tsx

```diff
@@ -110,6 +114,37 @@ const ProfileEdit: React.FC<IProfileEditProps> = (props) => {
         }
         wrap();
     }, [namespacesJson, cnQuotaOptionsJson, quotaSubmitRefresh]);
+    
+    //TODO implement interval to check editrequeststatus
+    useEffect(() => {
+        async function wrap() {
+            const editRequest = await api.getEditRequestStatus(profileId);
+            if (editRequest.data.length === 0){
+                try {
+                    setPendingEditRequest(false);
+                } catch (err) {
+                    promptErrToastWithText(err.message);
+                }
+            }
+        }
+        wrap();
+        // eslint-disable-next-line
+    }, []);
+
+    // start polling for profile provision status changes every 30s
+    useInterval(() => {
+        async function wrap() {
+            const editRequest = await api.getEditRequestStatus(profileId);
+            if (editRequest.data.length === 0){
+                try {
+                    setPendingEditRequest(false);
+                } catch (err) {
+                    promptErrToastWithText(err.message);
+                }
+            }
+        }
+        wrap();
+    }, 1000 * 30);
```

Comment:
I am not sure how the polling mechanism is intended here. OR is this still in working progres? Thanks.

Reply:
This PR is strictly for the WEB updates in an effort to avoid a gigantic PR like last sprint. Currently the API simply responds with a successful 201 response. So when the poll checks for any edits it won't find them so mark the contact available to edit.